.github/workflows/ci.yml

```
@@ -0,0 +1,23 @@
+name: CI
+
+on: [push, pull_request]
+
```

Oh and one nice cosmetic thing, this will colour the pytest output ([for example](https://github.com/hugovk/unittest2pytest/runs/3676430246?check_suite_focus=true)):

```suggestion
on: [push, pull_request]

env:
  FORCE_COLOR: 1
```

---

Nice - didn't know about this 👍🏼

---

https://github.com/inverse/py-filelock/runs/3676569440?check_suite_focus=true#step:5:15